ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -291,6 +298,21 @@ const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById })
     previousCall = { content: 'Identical Call', hover: 'Identical call in previous callset', color: 'blue' }
   } else if (genotype.prevOverlap) {
     previousCall = { content: 'Overlapping Call', hover: 'Overlapping call in previous callset', color: 'teal' }
+  } else if (Number.isInteger(genotype.prevNumAlt)) {
+    const hasSameHemiXGenotype = isHemiX && (genotype.numAlt === 1 || genotype.numAlt === 2) &&
+      (genotype.prevNumAlt === 1 || genotype.prevNumAlt === 2)
+    previousCall = {
+      content: 'New Genotype',
+      color: 'olive',
+      hover: (
+        <span>
+          Previous callset:
+          <AlleleContainer>{svNumAltGenotype(genotype.prevNumAlt, isHemiX)}</AlleleContainer>
+          {/* eslint-disable-next-line react/jsx-one-expression-per-line */}
+          {hasSameHemiXGenotype && <i>Hemi alt allele change: {genotype.prevNumAlt} to {genotype.numAlt}</i>}
```

Comment:
The variable name `hasSameHemiXGenotype` looks like something changed, but the displaying text says `allele changed`.

Reply:
the number of called alt alleles has changed